src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -1203,6 +1229,227 @@ TEST_F(ResumeCtrlTest, GetSavedAppHmiLevel_AskedAppFound_INVALID_ENUM) {
             res_ctrl_->GetSavedAppHmiLevel(kTestPolicyAppId_, kMacAddress_));
 }
 
+TEST_F(ResumeCtrlTest, ResumptiLowVoltage_AppInFull_Restored) {
```

Comment:
@ZhdanovP Check typo here please

Reply:
@yang1070 @AByzhynar the names were fixed at  [7f61588](https://github.com/smartdevicelink/sdl_core/pull/2418/commits/7f6158896248a86d2be85d5e10957bb0e55f0a5e)